content/en/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md

```diff
@@ -24,6 +24,12 @@ A [Kubernetes Ingress Resources](https://kubernetes.io/docs/concepts/services-ne
 
 Let's see how you can configure a `Ingress` on port 80 for HTTP traffic.
 
+1. First, deploy the `httpbin` application:
```

Comment:
This isn't needed... this is part of the "Before you begin" section and `Before you begin` link here: https://github.com/istio/istio.io/pull/13698/files#diff-8c80e56d7fd2f0d7a9c4a9f85d2ff0335283c9a284b82fe28e937f3419141031R19

Reply:
I missed it, since it's in a different page and at every end of page, I will follow up the clean command. Thanks, I will close this.